src/App.css

```diff
@@ -12,11 +12,128 @@
 .nav-container {
   display: flex;
   font-size: x-large;
-  gap: 19px;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 4.25rem;
+  background-color: #fff;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
 }
 
 .main-nav {
   gap: 22px;
   display: flex;
   list-style: none;
 }
+
+.main-nav a {
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.523);
+  list-style: none;
+}
+
+.main-title {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0241ff;
+}
+
+.img-png {
+  display: flex;
+  border: 2px solid rgb(234, 223, 223);
+  border-radius: 50%;
+  margin-right: 10px;
+  padding: 4px;
+  margin-left: auto;
+}
+
+.image {
+  height: 15px;
+  width: 20px;
+}
+
+.card {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 4fr 1fr 1fr;
+  justify-content: space-between;
+  align-items: center;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  gap: 5%;
+}
+
+.book--utils button {
+  padding: 0.5rem;
+  cursor: pointer;
+  width: 4.563rem;
+  height: 2.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #0241ff;
+  background: none;
+  border: none;
+}
+
+.chapter {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: max-content;
+}
+
+.loader {
+  width: 7.25rem;
+  height: 6.25rem;
+  color: #0241ff;
+}
+
+.oval {
+  color: #0241ff;
+}
+
+.submit {
+  padding: 17px;
+  background-color: #0241ff;
+  border: none;
+  color: white;
+}
+
+.b-color {
```

Comment:
- [ ] I suggest that you use colors that were used in the design like `#0290ff;` and `#4386bf;`